src/auth0/handlers/roles.js

```diff
@@ -121,4 +128,23 @@ export default class RoleHandler extends DefaultHandler {
       }
     }));
   }
+
+  async checkTenantSupportsRoles() {
+    try {
+      await this.client.roles.get({ id: 'rol_0000000000000000' });
```

Comment:
is there some way to asses role support aside from hardcoding a role id?

Reply:
No other way without actually making the call and to make the call some fake role id is required. role-id has some format - https://github.com/auth0/api2/blob/3415397d3534fff97f9cb4c442598b3831b88580/config/z-schema.js#L57 so `rol_0000000000000000` seemed like the best option